src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -2,12 +2,14 @@
 
   <PropertyGroup>
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
-
-    <!-- NuGet -->
     <Version>1.10.0</Version>
     <Title>Datadog APM - ClrProfiler</Title>
-    <Description>Automatic instrumentation library used by .NET Tracer for Datadog APM. Requires separate native library that implements the CLR Profiling API.</Description>
+    <Description>Automatic instrumentation library used by .NET Tracer for Datadog APM. Requires separate native library that implements the CLR Profiling API.</Description>	
```

Comment:
Whoops, by pasting in this description I undid your comment about this being obsolete. Can you place that back in this Description?

Reply:
Fixed in af9622aec9d2ce00e105506b149b32b36a40f4bb.